code/__defines/items_clothing.dm

```diff
@@ -1,6 +1,6 @@
 #define HUMAN_STRIP_DELAY        40   // Takes 40ds = 4s to strip someone.
 
-#define SHOES_SLOWDOWN          -1.0  // How much shoes slow you down by default. Negative values speed you up.
```

Comment:
what have you done with shoes?

I'm planning to overhaul movement speed in future, it shouldn't be touched with a broad ranging PR like this